src/main/kotlin/FilePathGenerator.kt

```diff
@@ -0,0 +1,103 @@
+import java.io.File
+import java.lang.IllegalArgumentException
+
+enum class SupportedExtensions(val ext: String) {
+    WAV("wav"),
+    MP3("mp3"),
+    JPEG("jpeg"),
+    JPG("jpg"),
+    TR("tr");
+
+    companion object {
+        private fun getSupportedContainers(): Array<SupportedExtensions> = arrayOf(TR)
+        private fun getCompressedTypes(): Array<SupportedExtensions> = arrayOf(MP3, JPEG, JPG)
+
+        fun isSupportedExtension(ext: String): Boolean = values().any { it.name == ext.toUpperCase() }
+
```

Comment:
Use line spacing to visually group functionality of similar intent. Similarly, don't be inconsistent about it. The method on line 15 fits with the kind of functionality/ purpose as lines 17 and 18. So don't put whitespace between them if you're going to have whitespace here at all.